### ISSUE TITLE
HDD and DVD needed?

### ISSUE THREAD

**Issue by Velocet** [Ryzee119/XboxDoomBIOS#4]
Are the HDD and DVD needed or is it possible to only have a usb thumb drive connected to play the game?

From my understanding the only change needed would be to alter the path?

**Comment by Ryzee119** [comment]
> From my understanding the only change needed would be to alter the path? 

Correct, it supports USB mass storage drives formatted as exFat or FAT32 with 'default cluster' size.

They are mounted automatically with drive letters starting from `0:` but its just hardcoded to `e:/doom` for no particular reason

I did most of my development before I wrote an ATA/ATAPI driver and used a USB stick

We need to implement a fall back if `e:/doom` doesnt find anything
